conference.js

```diff
@@ -2486,6 +2486,7 @@ export default {
 
                     // false because the thank you dialog shouldn't be displayed
                     .catch(() => Promise.resolve(false));
+            APP.API.notifyFeedbackPromptDisplayed();
```

Comment:
I wonder if this is best fired in the feedback modal itself. And maybe a close event too? Not sure if the close is needed for now.

Reply:
yeah, probably because it's a "maybe"

Reply:
Oh actually, maybe shoving this in middleware is best. There's a todo that I'm supposed to do where I put all the APP.API calls into middleware. I guess in the big picture it doesn't matter too much if it goes into middleware now or later.

Reply:
Do you mean a middleware which will listen for the openFeedbackDialog action ?

Reply:
Yeah. I’ve been meaning to make a web-only middleware so that we can start calling out to APP.API from it and it alone, instead of calling it from all around the codebase. So this middleware could listen to actions and sniff out feedback open and close and call APP.API as needed, but...adding the APP.API calls to the feedback maybe good more than enough for now.